code/index.html

```diff
@@ -1,22 +1,216 @@
+    rel="stylesheet">
+</head>
+
+<body>
+
+
+  <!-- SECTION 1 - logo & navbar -->
+  <header>
+    <section id="section-1" class="section-1">
+
+      <nav class="nav-top">
+        <a class="logo-container" href="#">
+          <img class="logo" src="images/logo/logo-4.png" alt="Logo image">
+        </a>
+        <h1 class="site-name-top">pattern news</h1>
+        <a class="nav-bar" href="#">trends</a>
+        <a class="nav-bar" href="#">print</a>
+        <a class="nav-bar" href="#">designs</a>
+        <a class="nav-bar" href="#">collections</a>
+      </nav>
+
     </section>
+  </header>
+
+
+  <!-- SECTION 2 - the big news content -->
+  <section id="section-2" class="section-2">
+
+    <figure class="img-container"><img src="images/news-images/news-39.jpg" alt="Pattern news"></figure>
+    <div class="big-news">
+      <a class="news" href="#">
+        <h2>Botanical colonies – the latest trend</h2>
+        <p>With prints of everything from mushrooms, plants and verdant green leaves to picturesque palm trees and
+          exotic flowers, the botanical patterns provide you with endless opportunities for creating that tropical
+          garden vibe. In the botanical prints category you will find both patterns with a vintage atmosphere, patterns
+          with a vivid floral expression as well as patterns with a modern graphic design.</p>
+        <p class="read-more">[Read more] &rarr;</p>
+      </a>
+    </div>
+
+  </section>
+
+
+  <!-- SECTION 3 – the other news -->
+  <section id="section-3" class="section-3">
+
+    <!-- CARD - 1 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-01.jpg" alt="multi color paint">
+          <h3 class="text-on-pic">cool stuff</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Paint</span> - more is more.</p>
+    </div>
+
+    <!-- CARD - 2 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-49.jpg" alt="the desert">
+          <h3 class="text-on-pic">the desert</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Lost</span> in sandstorm.</p>
+    </div>
+
+    <!-- CARD - 3 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-33.jpg" alt="magenta neon lights">
+          <h3 class="text-on-pic">neon lights</h3>
+        </a>
+      </div>
+      <p class="card-text"><span class="card-text-highlight">Awake</span> at nights.</p>
+    </div>
+
+    <!-- CARD - 4 -->
+    <div class="card-container">
+      <div class="card">
+        <a href="#">
+          <img class="img-card" src="images/news-images/news-23.jpg" alt="yellow tulips">
```

Comment:
Awesome that you've structured your files and used an image folder and a HUGE plus for your use of the alt in the image tag 👏 BRAVO!